app/cypress/support/listAuthenticatedPages.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs').promises;
+const path = require('path');
+
+async function listAuthenticatedPages(
```

Comment:
should this be a cypress command instead, called at the beginning of the login-redirect spec file?

Reply:
I tried initially going that route, but unfortunately there are a few constraints that prevented it:
- `cy.exec` returns limited data, but I did try a solution writing to `process.stdout` and then parsing stdout in the command
- however, the scope of env variables dynamically set with `Cypress.env` is limited to within individual tests (if set in the command scope, then not present in the test)
  * even when the command is written in the spec file, the variable is used _to generate_ tests and is therefore needed to be scoped _outside_ a test.
  * cannot run a Cypress command outside of a test. Even running the command in a different, fake test in the same specfile does not prioritize the fake test to run before the loop that generates the real tests.

As result of that exploration, the written file seems to be necessary.